webpack.config.js

```diff
@@ -8,9 +8,11 @@ module.exports = [
     output: {
       path: path.join(__dirname, 'dist'),
       filename: 'teddy.js',
-      library: 'teddy',
-      libraryTarget: 'umd',
-      libraryExport: 'default',
```

Comment:
`libraryTarget` and `libraryExport` are being deprecated soonish - the docs recommend the changes in the diff.